Navigation Breadcrumb For Officer Page

fritzdavenport opened the pull request:
## Status

Ready for review / in progress

## Description of Changes

Fixes #582 

Changes proposed in this pull request:

 - Navigation Breadcrumbs for Office Page

## Notes for Deployment

Nuns.

## Screenshots (if appropriate)

![breadcrumbs](https://user-images.githubusercontent.com/5983782/50061707-ae13aa00-0170-11e9-9c3e-3991f30aac8c.png)

## Tests and linting

 - [x] I have rebased my changes on current `develop`

 - [x] pytests pass in the development environment on my local machine

 - [ ] `flake8` checks pass - broken on master???

redshiftzero commented on OpenOversight/app/templates/officer.html:
```diff
@@ -4,7 +4,7 @@
 
   <div class="page-header">
-    <h1>Officer Detail: <b>{% include 'partials/officer_name.html' %}</b></h1>
+      <h1><a href={{url_for('main.list_officer',department_id=officer.department.id)}}>{{ officer.department.name|title }}</a> >> <b>{% include 'partials/officer_name.html' %}</b></h1>
```
Looks good! So one idea: there are styled breadcrumbs in bootstrap (which we use) - see [here](https://getbootstrap.com/docs/3.3/components/#breadcrumbs), want to try using those? They look pretty nice

fritzdavenport commented:
:+1: @redshiftzero, updated. Also added a `name` field 
![breadcrumbs](https://user-images.githubusercontent.com/5983782/50062934-cfc45f80-0177-11e9-8725-3c82d604c605.png)

redshiftzero commented:
One last request: Can we preserve the title in the `<h1>` tag and put the breadcrumb below it? That _should_ also resolve the test failures

fritzdavenport commented:
@redshiftzero  fixed
![screenshot from 2018-12-17 01-59-52](https://user-images.githubusercontent.com/5983782/50071383-956eb880-019f-11e9-90f1-709c150d953d.png)
